_data/file_formats/eff_v1/header.yml

```diff
@@ -53,61 +53,67 @@
+      - At <code><a href="../ie_formats/gam_v2.0.htm#GAMEV2_0_Header_GameTime">Gametime</a>(ticks) >= <a href="#effv1_Header_0xE">Duration</a>(ticks)</code> converts to <code><a href="#effv1_Header_0xC_1">Instant/Permanent</a></code>, <code><a href="../ie_formats/eff_v2.htm#effv2_Body_0x28">new duration</a>(ticks) = 16 * <a href="#effv1_Header_0xE">Duration</a>(ticks)</code>.
         - (If effect applied directly with <code><a href="#effv1_Header_0xC_7">Permanent after duration</a></code>)
-        - It will convert to <code><a href="#effv1_Header_0xC_1">Instant/Permanent</a></code> at the same `Gametime(ticks)` no matter when the effect was applied.
+        - It will convert to <code><a href="#effv1_Header_0xC_1">Instant/Permanent</a></code> at the same <code><a href="../ie_formats/gam_v2.0.htm#GAMEV2_0_Header_GameTime">Gametime</a></code> (ticks) no matter when the effect was applied.
+
     - 8 ⟶ <b><a name="effv1_Header_0xC_8">Equipped after duration</a></b>
       - This timing mode measures <code><a href="#effv1_Header_0xE">Duration</a></code> in ticks (<small><sup>1</sup>⁄<sub>15</sub></small> second).
-      - At <code>Gametime(ticks) >= <a href="#effv1_Header_0xE">Duration</a>(ticks)</code> converts to <code><a href="#effv1_Header_0xC_2">Instant/While equipped</a></code>, <code>new duration(ticks) = irrelevant</code>.
+      - At <code><a href="../ie_formats/gam_v2.0.htm#GAMEV2_0_Header_GameTime">Gametime</a>(ticks) >= <a href="#effv1_Header_0xE">Duration</a>(ticks)</code> converts to <code><a href="#effv1_Header_0xC_2">Instant/While equipped</a></code>, <code><a href="../ie_formats/eff_v2.htm#effv2_Body_0x28">new duration</a>(ticks) = irrelevant</code>.
+
     - 9 ⟶ <b><a name="effv1_Header_0xC_9">Instant/Permanent</a></b> (after Death)
       - Effect is applied instantly.
-      - Never expires, but can terminate or be removed. <code>New duration(ticks) = <a href="#effv1_Header_0xE">Duration</a>(ticks)</code>.
+      - Never expires, but can terminate or be removed. <code><a href="../ie_formats/eff_v2.htm#effv2_Body_0x28">New duration</a>(ticks) = <a href="#effv1_Header_0xE">Duration</a>(ticks)</code>.
+
     - 10 ⟶ <b><a name="effv1_Header_0xC_10">Instant/Limited</a></b>
       - This timing mode measures <code><a href="#effv1_Header_0xE">Duration</a></code> in ticks (<small><sup>1</sup>⁄<sub>15</sub></small> second).
-      - Instantly converts to <code><a href="#effv1_Header_0xC_4096">Absolute duration</a></code>, <code>new duration(ticks) = Gametime(ticks) + <a href="effv1_Header_0xE">Duration</a>(ticks)</code>.
+      - Instantly converts to <code><a href="../ie_formats/eff_v2.htm#effv2_Body_0x24_4096">Absolute duration</a></code>, <code><a href="../ie_formats/eff_v2.htm#effv2_Body_0x28">new duration</a>(ticks) = <a href="../ie_formats/gam_v2.0.htm#GAMEV2_0_Header_GameTime">Gametime</a>(ticks) + <a href="#effv1_Header_0xE">Duration</a>(ticks)</code>.
```

Comment:
Can you please choose one and make it the main carrier of info? Now we link both to v2 from v1 and from v1 to v2.

Reply:
> Can you please choose one and make it the main carrier of info? Now we link both to v2 from v1 and from v1 to v2.

- The main carrier of info is `EFF V1`. I just moved `Absolute Duration` from `EFF V1` to `EFF V2` because you cannot set `timing_mode` to `4096` (since `EFF V1 size = 1 byte`).
- `New duration` points to the `EFF V2 duration` field since embedded effects (`EFF V1`) get ultimately stored as `EFF V2` effects.
- I added a link to the `Gametime` field in the `GAM V2.0` file format.
- There are links from `v2` to `v1` specifically because `v1` is the main carrier of info

Am I missing something...?

Reply:
ok, then, carry on

Reply:
> ok, then, carry on

Feel free to merge, this PR was supposed to tweak just `op20` and `op232`...

Reply:
The EVILCUTOFF changes still need to be reverted though, that's what I'm waiting for.

Reply:
> The EVILCUTOFF changes still need to be reverted though, that's what I'm waiting for.

Nope, they appear to be correct...?

Unless I'm missing something, Bubb specifically said that (see the link I posted above):
- `Target=2 => NearestEnemyOf()` *which evaluates as **[EVILCUTOFF]**
- `Condition=1 => See(NearestEnemyOf())` *which evaluates as See(**[EVILCUTOFF]**)

Reply:
Somehow I missed that you changed them back, my bad.

Reply:
> Somehow I missed that you changed them back, my bad.

Cheers!